Cannot run the backend: "An exception occured while executing the Java class. io.github.agathevaisse.DictionaryApp"

Hello,

I have just created my Lexicala account. However, I am not able to run `penda`'s backend: 
```shell
$ mvn --version
Apache Maven 3.6.3 (cecedd343002696d0abb50b32b541b8a6ba2883f)
Maven home: /usr/local/Cellar/maven-deluxe/3.6.3-0/libexec
Java version: 11.0.8, vendor: AdoptOpenJDK, runtime: /Users/nicolas/.sdkman/candidates/java/11.0.8.hs-adpt
Default locale: en_FR, platform encoding: UTF-8
OS name: "mac os x", version: "10.15.7", arch: "x86_64", family: "mac"

$ export LEXICALA_USERNAME=******** ; ./run.sh                                                                                                                        
[ERROR] Failed to execute goal org.codehaus.mojo:exec-maven-plugin:3.0.0:java (default-cli) on project dictionary-api: An exception occured while executing the Java class. io.github.agathevaisse.DictionaryApp -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException

$ git log -1
commit ab6bcb8b23ec6ba8d7d8723ed34f5784b4da0cfb (HEAD -> main, origin/main, origin/HEAD)
Author: dependabot-preview[bot] <27856297+dependabot-preview[bot]@users.noreply.github.com>
Date:   Sun Oct 25 14:19:10 2020 +0000

    Bump ts-node from 8.3.0 to 9.0.0 in /penda

    Bumps [ts-node](https://github.com/TypeStrong/ts-node) from 8.3.0 to 9.0.0.
    - [Release notes](https://github.com/TypeStrong/ts-node/releases)
    - [Commits](https://github.com/TypeStrong/ts-node/compare/v8.3.0...v9.0.0)

    Signed-off-by: dependabot-preview[bot] <support@dependabot.com>
```
PS: my Lexical account is not "******". 😉 

Any tip? 😊

--- a/run.sh
+++ b/run.sh
@@ -2,4 +2,4 @@
 
 set -eu
 
-mvn --quiet exec:java -Dexec.mainClass=io.github.agathevaisse.DictionaryApp
+mvn --quiet verify exec:java "$@" -Dexec.mainClass=io.github.agathevaisse.DictionaryApp

